spec.bs

```diff
@@ -888,9 +890,9 @@ A <dfn for=fencedframetype>destination event</dfn> is either a
         1. Otherwise, set |attributionReportingEligibility| to "<code>[=eligibility/event-source=]</code>".
 
         1. Set |processResponse| to these steps given a [=response=] |response|:
-
+           1. Let |fenced| be true.
            1. Run [=process an attribution eligible response=] with |event|'s
-              [=destination enum event/attributionReportingContextOrigin=], |attributionReportingEligibility|, and |response|.
+              [=destination enum event/attributionReportingContextOrigin=], |attributionReportingEligibility|, |fenced|, |request|'s [=request/trigger verification metadata=], and |response|.
```

Comment:
Where is the `|request|` variable here? Is it in scope? Do we have access to it?

Reply:
I'm not sure what's the best way to spec this. `|request|` is defined a few lines below and this is the function to process the response of the request.

Reply:
At first I considered recommending you to define request higher up (but keep it empty), so that we could at least reference it here. The implicit behavior here would be that by the time _processBody_ finally runs, the `|request|` it accesses is not the old stale empty one that is in scope, but is instead the version of `|request|` that has gone through the Fetch infra and had its `trigger verification metadata` populated etc.

But that doesn't feel very rigorous to me, so I guess I would rather see the attribution reporting spec monkeypatch Fetch in way that transfers over the `trigger verification metadata` bit from the request to the response, if you really need to access it in the response during _processBody_. That's probably the cleanest & most rigorous way. It kind of looks like that's how we do it in the implementation but I lost track of tracing all of the associated callbacks back from this point: https://source.chromium.org/chromium/chromium/src/+/main:services/network/attribution/attribution_request_helper.cc;l=315;drc=63635fff4bce239256a42d5f1a28fa3d0934bf35.

Reply:
Thanks Dom. That sounds reasonable. cc @agarant.

In the meantime, since fenced frame beacons are only eligible for sources and therefore trigger verification metadata is irrelevant. Updated to hardcode it to null instead. Thanks.

Reply:
This is indeed how the implementation works, I can look into making this change.

Reply:
Thanks. I'd be most comfortable landing this after that ARA spec change is fixed, or possibly by putting a prominent TODO around the hardcoding that you added (something [like this](https://wicg.github.io/fenced-frame/#issue-0ffad658)). I prefer the former though, and keeping this thread up to date on the ARA spec progress there.

Reply:
Hi @domfarolino , we've removed trigger verification from ARA spec. Could you please review this PR again, thanks!